ja/orm/retrieving-data-and-resultsets.rst

```diff
@@ -368,7 +368,7 @@ fineder メソッドは、あなたが作成したい finder の名前が ``Foo`
     }
 
     // コントローラーやテーブルのメソッド内で
-    // Prior to 3.6 use TableRegistry::get('Articles')
+    // 3.6 より前は、 TableRegistry::get('Articles') を使用
```

Comment:
This comment is for 3.x. Please change direction to 3.x branch.

Reply:
Or, please try translating [here](https://github.com/cakephp/docs/blob/4.x/en/orm/retrieving-data-and-resultsets.rst).

Reply:
Please remove the "prior to 3.x" code in 4.x docs.

Reply:
@okinaka @othercorey Thank you for reviewing and commenting.
I did add commit with remove description of 3.x.